source/aks/patching-artifactory.html.md.erb

```diff
@@ -1,152 +1,94 @@
 ---
 title: Artifactory Patching
 weight: 30
-last_reviewed_on: 2024-03-13
+last_reviewed_on: 2024-06-06
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
-Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. Here's how it is used based on the provided instructions:
+Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment.
 
-The steps below will show you how to check the current version of Artifactory being used (7.71.5) and then to update it to a newer version (7.77.3). This process involves accessing Artifactory's website to determine the latest version available.
+The steps below will show you how to check and update **self-hosted version** of Artifactory being used. This process involves accessing [Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes) to determine the latest **self-hosted version** version available.
 
-Before applying the update, a testing process is initiated. This involves downloading a repository containing configuration files for Artifactory (artifactory-sbox.yaml). This file is created and modified to reflect the desired changes, such as updating the version number.
+Before applying the update on PTL AKS Cluster, a testing process is initiated and is conducted on PTLSBOX AKS Cluster. This involves raising a PR to [GitHub](https://github.com/hmcts/cnp-flux-config) containing configuration files for Artifactory, creating 'artifactory-sbox.yaml' file and editing 'kustomization.yaml' files with your changes as seen in the example PR below.
 
-Kubernetes configuration files (kustomization.yaml) are updated to include the changes made in the testing phase. This ensures that the updated configuration is applied correctly within the Kubernetes cluster.
+[Exampe PR](https://github.com/hmcts/cnp-flux-config/pull/32185/files) of PTLSBOX changes
 
-Changes made to the configuration files are submitted for review through a pull request (PR) process. Once approved, the changes are merged into the main branch, allowing them to be deployed.
+[Exampe PR](https://github.com/hmcts/cnp-flux-config/pull/32191/files) of PTL changes after testing on PTLSBOX
 
 If any issues arise during the update process, troubleshooting steps are provided. These steps involve identifying and resolving any errors or conflicts that may occur, such as missing namespaces or configuration discrepancies.
 
-# Artifactory Patching Process
+## Artifactory Patching Process
 
-1. Azure CLI Login and Kubernetes Services:
+### 1. Connect to CFT PTLSBOX AKS Cluster
 
-• Access Azure CLI and search for Kubernetes Services
+  • [Connect to PTLSBOX cluster](https://hmcts.github.io/cloud-native-platform/troubleshooting/index.html#connecting-to-aks-clusters)
 
-• Type the following PTL in the search box:
+### 2. Check Artifactory Pods
```

Comment:
Is there a purpose of this step before updating pltsbox? Maybe needs to say check they and the helm release is healthy before starting an update?

Reply:
Otherwise I may think "Why am I doing this step"

Reply:
Tbh there is no real reason other than verifying previous version is there. Will remove because it adds no real value to the overall doc.